packages/additionalapi/src/database/AdditionalQueries.ts

```diff
@@ -26,7 +20,7 @@ export class AdditionalQueries {
      * @param nodeIdList
      * @param depthLimit
      */
-    getNodeTree = async (nodeIdList: string[], depthLimit: number): Promise<QueryReturnType<NodeTreeResultType[]>> => {
+    getNodeTree = async (nodeIdList: string[], depthLimit: number): Promise<QueryReturnType<NodeTreeResultType[] | string>> => {
```

Comment:
Could it make sense to define an alias for `NodeTreeResultType[] | string`, something like "QueryResultOrError"? This would make more obvious to see the role of the string value

Reply:
Removed the `| string` and not catch the exception here.  It will be caught by the generic catch-all.